pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/LocalMemoryMetadataStore.java

```diff
@@ -84,6 +88,14 @@ public LocalMemoryMetadataStore(String metadataURL, MetadataStoreConfig metadata
             // Use a reference from a shared data set
             String name = uri.getHost();
             map = STATIC_MAPS.computeIfAbsent(name, __ -> new TreeMap<>());
+            STATIC_INSTANCE.compute(name, (key, value) -> {
+                if (value == null) {
+                    value = new HashSet<>();
+                }
+                value.forEach(v -> registerListener(v));
```

Comment:
Should this be `registerListener(this)` ?

Reply:
+1

Reply:
No need, The parent class has already been registered.

https://github.com/apache/pulsar/blob/c9f68f09dd43f92214b0b2f14dae2e24b2afb4eb/pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/AbstractMetadataStore.java#L77-L80

Reply:
After a second thought, this should be like this?
```
v. registerListener(this); 
this.registerListener(v);
```

Reply:
@Jason918 We can discuss whether to share the listener in the whole JVM.

Reply:
> After a second thought, this should be like this?
> 
> ```
> v. registerListener(this); 
> this.registerListener(v);
> ```

Sorry, I fix.